.github/workflows/main.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        python-version: ['3.8', '3.9', '3.10', '3.11', '3.12']
+        python-version: ['3.9', '3.10', '3.11', '3.12']
```

Comment:
I see on qiskit release notes

> The minimum supported Python version is now 3.9 as Python 3.8 went end of life in 2024-10. Official support for Python 3.13 support was also added in this release.

https://docs.quantum.ibm.com/api/qiskit/release-notes/1.3#prelude-1

Do you think we can add support to 3.13 as well ?

Reply:
Let's wait for qiskit-aer to support python 3.13 on windows as well https://github.com/Qiskit/qiskit-aer/issues/2277